### ISSUE TITLE
Chrome config - how to set it up properly?

### ISSUE THREAD

**Issue by llinfeng** [evilmanimani/iswitchw#5]
Please advise if I am missing something. Here is what I see in `chrome://accessibility` after following the instruction in ReadMe. However, all tabs in Chrome are still missing from the `iswitchw` window.
> (go) to chrome://accessibility and checking every option under "Accessibility modes:" on the left side of the page.
![image](https://user-images.githubusercontent.com/5890407/169846762-b631f0dc-e853-43ea-86d1-81b77bce82c6.png)

I've tried to run `iswitchw.ahk` both with and without admin privilege, and the Chrome window in the screenshot is not running as admin. I started it with the `--force-renderer-accessibility` flag, which happens to check everything for me.

Am I missing something simple in `chrome://accessibility`? If not, is there some setup I can do to get `iswitchw` list Chrome tabs? Thanks a lot!

**Comment by mcleodd10** [comment]
Hey sorry didn't see this right away; I'm sure you've moved on at this point, but besides, I haven't checked in on it in a while, but yeah I think something changed with the accessibility with Chrome since I can't get it to work now either; I'll give it a look when I get a chance.

**Comment by mcleodd10** [comment]
Should be working now with the latest updated I just pushed, details on setup are in the readme and in the ahk file as well.
